nemo_skills/finetuning/data_preparation_utils/sdp_configs/base_config.yaml

```diff
@@ -0,0 +1,51 @@
+processors_to_run: all
```

Comment:
It will be more user-friendly if most important parameters for different processors are available as top-level arguments. E.g. create a new argument `prediction_jsonl_files` and then in the first processor use it directly via `${prediction_jsonl_files}`. Same for output location or important filtering steps, like downsampling_method, etc. Basically, most of the parameters that we currently have in the original prepare script can be used here for convenience. You can also use boolean conditions to skip certain processors based on top-level arguments like here https://github.com/NVIDIA/NeMo-speech-data-processor/blob/main/dataset_configs/italian/mls/config.yaml#L102 or here https://github.com/NVIDIA/NeMo-speech-data-processor/blob/main/dataset_configs/italian/mls/config.yaml#L196

Reply:
If you find it convenient to skip processors via top-level args, you can potentially try to combine code/non-code flow into a single config and just pick different subparts based on the args. But creating multiple configs might be easier